plugin.audio.podcatcher/changelog.txt

```diff
@@ -0,0 +1,34 @@
+version 0.2.1 - Change to Request-lib to get more stable results
```

Comment:
Please drop from the submission. This is not used anymore in Kodi in favor of the `<news>` tag